packages/frontend/amp/components/Byline.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { bylineTokens } from '@frontend/amp/lib/byline-tokens';
+import { css } from 'emotion';
+import { headline } from '@guardian/pasteup/typography';
+import { pillarPalette } from '@frontend/lib/pillars';
+
+const bylineStyle = (pillar: Pillar) => css`
+    ${headline(2)};
+    color: ${pillarPalette[pillar].main};
+    padding-bottom: 8px;
+    font-style: italic;
+
+    a {
```

Comment:
More of query after merge - do we have rules around nesting? I prefer to be explicit, creating a seperate `const` for styles and applying directly.

Reply:
Yeah, generally our rule is to avoid nesting, though one-level is considered okay.